unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Scripts/ControlsModule/SliderControlModel.cs

```diff
@@ -15,9 +15,6 @@ public class SliderControlModel : SettingsControlModel
         [Tooltip("Max allowed value for the slider.")]
         public float sliderMaxValue;
 
-        [Tooltip("True if the slider values will be integers.")]
-        public bool sliderWholeNumbers;
```

Comment:
A consequence of remove this param is that the sliders that we had using it, now will show their values as decimal, like en these examples:

![image](https://user-images.githubusercontent.com/64659061/114210038-9a069e80-995f-11eb-8ee3-5bbd7989885a.png)

We should fix it.